takari-lifecycle-plugin-its/src/test/java/io/tesla/maven/plugins/test/DeployAtEndTest.java

```diff
@@ -9,7 +9,7 @@
 import io.takari.maven.testing.executor.MavenRuntime.MavenRuntimeBuilder;
 import io.takari.maven.testing.executor.MavenVersions;
 
-@MavenVersions({"3.3.9", "3.5.0"})
+@MavenVersions({"3.6.3"})
```

Comment:
Can you confirm your intention is to desupport all earlier Maven versions with this PR?